packages/teams-js/test/public/call.spec.ts

```diff
@@ -22,13 +22,17 @@ describe('call', () => {
     }
   });
 
-  it('should not allow calls before initialization', () => {
-    return expect(call.startCall(mockStartCallParams)).rejects.toThrowError('The library has not yet been initialized');
+  it('should not allow calls before initialization', async () => {
+    expect.assertions(1);
+    await call
+      .startCall(mockStartCallParams)
+      .catch(e => expect(e).toMatchObject(new Error('The library has not yet been initialized')));
   });
 
-  it('shoud not allow calls if not supported', () => {
+  it('shoud not allow calls if not supported', async () => {
+    expect.assertions(1);
```

Comment:
Silly question, is there any reason why in one test you use `.toMatchObject` and another you use `.toEqual`

Reply:
I know from just having looked it up, that they are ever so slightly different

Reply:
i think they'd both be fine, i'm just curious

Reply:
Good call, we should stick to one for consistency. Just taking a quick look, `.toEqual` seems more correct for what we're testing, but I think we could go either way